services/graph/pkg/config/config.go

```diff
@@ -31,7 +31,9 @@ type Config struct {
 type Spaces struct {
 	WebDavBase                      string `yaml:"webdav_base" env:"OCIS_URL;GRAPH_SPACES_WEBDAV_BASE" desc:"The public facing URL of WebDAV."`
 	WebDavPath                      string `yaml:"webdav_path" env:"GRAPH_SPACES_WEBDAV_PATH" desc:"The WebDAV subpath for spaces."`
-	DefaultQuota                    string `yaml:"default_quota" env:"GRAPH_SPACES_DEFAULT_QUOTA" desc:"The default quota in bytes."`
+	DefaultQuota                    string `yaml:"default_quota" env:"GRAPH_SPACES_DEFAULT_QUOTA" desc:"The default quota for project spaces in bytes. Also understands common abbreviations such as '100KB', '300KiB', '1024MiB', '4GB', ..."`
+	MaxQuotaPersonal                string `yaml:"max_quota_personal" env:"GRAPH_SPACES_MAX_QUOTA_PERSONAL" desc:"The maximum quota for a personal space in bytes. Also understands common abbreviations such as '100KB', '300KiB', '1024MiB', '4GB', ..."`
+	MaxQuotaProject                 string `yaml:"max_quota_project" env:"GRAPH_SPACES_MAX_QUOTA_PROJECT" desc:"The maximum quota for a project space in bytes. Also understands common abbreviations such as '100KB', '300KiB', '1024MiB', '4GB', ..."`
```

Comment:
I liked the example string. Full listing of the abbreviations makes sense. But can we add one example additionally, so that admins can read the `syntax` by example? (I know, it's not hard, but an example makes me feel safer as a consumer of this).

Reply:
Added